spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -3,95 +3,69 @@
 include Features::ActiveJobHelper
 
 feature 'Two Factor Authentication' do
-  describe 'When the user has not setup 2FA' do
-    scenario 'user is prompted to setup two factor authentication at first sign in' do
-      sign_in_before_2fa
+  describe 'When the user has not set up 2FA' do
+    scenario 'user is prompted to set up two factor authentication at account creation' do
+      user = sign_in_before_2fa
+
+      # user attempts to circumnavigate 2FA setup
+      visit profile_path
```

Comment:
thoughts on throwing this in a method call "attempt_to_circumnavigate_2fa` or something rather than comment? I can imagine comment getting out of sync with code below more easily than a well-named method.